WooCommerce/Classes/Authentication/Store Creation/Profiler/StoreCreationProfilerQuestionContainerViewModel.swift

```diff
@@ -55,15 +57,19 @@ final class StoreCreationProfilerQuestionContainerViewModel: ObservableObject {
 
     @Published private(set) var currentQuestion: StoreCreationProfilerQuestion = .sellingStatus
 
-    init(storeName: String,
+    init(siteID: Int64,
+         storeName: String,
          stores: StoresManager = ServiceLocator.stores,
          analytics: Analytics = ServiceLocator.analytics,
          onCompletion: @escaping () -> Void,
-         uploadAnswersUseCase: StoreCreationProfilerUploadAnswersUseCaseProtocol) {
+         uploadAnswersUseCase: StoreCreationProfilerUploadAnswersUseCaseProtocol,
+         themeInstaller: ThemeInstallerProtocol? = nil) {
```

Comment:
❓ Why does this have to be optional and nil by default? Can we inject the default installer instead?

```suggestion
         themeInstaller: ThemeInstallerProtocol = DefaultThemeInstaller()) {
```

Reply:
Done in 36d0d57b7e121bbd4d9a3f7304c79d9871c07576